### PR TITLE
updates readme

### PR DESCRIPTION
**Pull request by jpvajda**
- adds maintainers 
- adds a section on version support for Apollo Client
- Fixes a few markdown linter issues.

### CODE REVIEW

**Comment by jpvajda** [comment]
@MrDoomBringer let me know what you wanted to add to this PR! Thanks!

File: README.md [hunk 1]
```diff
@@ -20,6 +20,23 @@ The Apollo Client Browser Devtools appear as an "Apollo" tab in your web browser
 
 ![Apollo Client Browser Devtools](./assets/ac-browser-devtools-3.png)
 
+## Apollo Maintainers
```
Comment (bignimbus) [inline README.md:23]:
```suggestion
## Maintainers
```

Can we drop the "Apollo" here?  I think one interpretation could be that we're the "Apollo Maintainers" for the company and I want to make sure we're setting expectations accordingly

File: README.md [hunk 2]
```diff
@@ -20,6 +20,23 @@ The Apollo Client Browser Devtools appear as an "Apollo" tab in your web browser
...
+- [@bignimbus](https://github.com/bignimbus)
+- [@jpvajda](https://github.com/jpvajda)
+- [@mrdoombringer](https://github.com/mrdoombringer)
+- [@jerelmiller](https://github.com/jerelmiller)
+
+## Apollo Client version support
+
+> if you are using an older version of Apollo Client and have issues with our Client Browser Devtools we recommend you upgrade to the latest version of Apollo Client.
+
+- We provide active support for the most current release of [Apollo Client](https://github.com/apollographql/apollo-client) for use with our Client Browser DevTools.
```
Comment (bignimbus) [inline README.md:36]:
```suggestion
- We provide active support for the current minor release of [Apollo Client](https://github.com/apollographql/apollo-client) for use with our Client Browser DevTools.
```

**Review by bignimbus — APPROVED** [review]
LGTM - added some suggestions @jpvajda, feel free to ignore them if they don't line up with what you were thinking 🙏🏻

**Comment by jpvajda** [comment]
@bignimbus   @MrDoomBringer  There seems to be failing tests on this, which was just a readme update. Any ideas what might be going on here?

**Comment by MrDoomBringer** [comment]
@jpvajda Yup! It's something I'm actually looking into right now - tests are failing as a result of the react-graphql-syntax-highlighter dependency change

**Review by MrDoomBringer — APPROVED** [review]

**Pull request merged**